osf/models/mixins.py

```diff
@@ -624,6 +624,18 @@ def get_reviewable_state_counts(self):
         counts.update({row['machine_state']: row['count'] for row in qs if row['machine_state'] in counts})
         return counts
 
+    def get_request_state_counts(self):
+        # import stuff here to get around circular imports
+        from osf.models import PreprintRequest
+        qs = PreprintRequest.objects.filter(target__provider__id=self.id,
+                                            target__node__isnull=False,
+                                            target__node__is_deleted=False,
+                                            target__node__is_public=True)
+        qs = qs.values('machine_state').annotate(count=models.Count('*'))
+        counts = {state.value: 0 for state in DefaultStates}
```

Comment:
Should this be `ReviewStates`?

Reply:
`ReviewStates` contains `withdrawn`. However, this is the state of `PreprintRequests`, which does not have a `withdrawn` state.